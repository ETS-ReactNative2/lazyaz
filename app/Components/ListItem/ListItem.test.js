import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { ListItem } from './index';

test('Chip component renders correctly', () => {
  const props = {
    text: '',
    image: 0,
  };
  const tree = renderer.create(<ListItem {...props} onPress={() => {}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
