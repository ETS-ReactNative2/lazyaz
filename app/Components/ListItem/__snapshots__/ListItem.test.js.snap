// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip component renders correctly 1`] = `
<View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text>
      
    </Text>
    <Image
      source={0}
    />
  </View>
  <View
    style={
      Array [
        undefined,
        Object {
          "backgroundColor": undefined,
          "marginVertical": 20,
        },
      ]
    }
  />
</View>
`;
